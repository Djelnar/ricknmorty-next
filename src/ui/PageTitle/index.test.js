/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import { PageTitle } from './index'


describe('PageTitle', () => {
  it('should render correctly', () => {
    const output = renderer.create(<PageTitle>Mock Content</PageTitle>).toJSON()

    expect(output).toMatchSnapshot()
  })
})
