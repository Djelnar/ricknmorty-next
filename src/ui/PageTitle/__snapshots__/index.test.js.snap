// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle should render correctly 1`] = `
.c0 {
  padding: 16px 8px 8px;
  font-size: 24px;
  color: #333333;
}

<h1
  className="c0"
>
  Mock Content
</h1>
`;
